pkg/log/log.go

```diff
@@ -162,8 +210,69 @@ func (mf *MyFormatter) Format(entry *logrus.Entry) ([]byte, error) {
 		funcName = funcName[subIndex+1:]
 	}
 
-	b.WriteString(fmt.Sprintf("%s:[%s %s::%s:%d]  %s\n",
-		entry.Time.Format("2006-01-02 15:04:05 -0700"), level, fileName, funcName, entry.Caller.Line,
+	b.WriteString(fmt.Sprintf("%s:[%s %s:#%d]  %s\n",
+		entry.Time.Format("2006-01-02 15:04:05 -0700"), level, fileName, entry.Caller.Line,
 		entry.Message))
 	return b.Bytes(), nil
 }
+
+func Fatalf(format string, args ...interface{}) {
+	dash.Fatal(format, args...)
+	tpLog.Fatalf(format, args...)
+}
```

Comment:
Just wondering if we could have done this by adding us as a hook under logrus. Perhaps, InfoD could have been implemented by using WithFields()?